File: src/com/sofps/inspirationalquotes/Quote.java
package com.sofps.inspirationalquotes;

import java.io.Serializable;

public class Quote implements Serializable {
	private static final long serialVersionUID = 1L;

	private long mId;
	private String mText;
	private String mAuthor;
	private int mTimesShowed;
	private String mLanguage;

	public Quote() {
		mId = -1;
	}

	public long getId() {
		return mId;
	}

	public void setId(long id) {
		mId = id;
	}

	public String getText() {
		return mText;
	}

	public void setText(String text) {
		mText = text;
	}

	public String getAuthor() {
		return mAuthor;
	}

	public void setAuthor(String author) {
		mAuthor = author;
	}

	public int getTimesShowed() {
		return mTimesShowed;
	}

	public void setTimesShowed(int timesShowed) {
		mTimesShowed = timesShowed;
	}

	public String getLanguage() {
		return mLanguage;
	}

	public void setLanguage(String language) {
		mLanguage = language;
	}

}

File: app/src/main/java/com/sofps/inspirationalquotes/data/Quote.kt
package com.sofps.inspirationalquotes.data

import java.io.Serializable

class Quote : Serializable {

    companion object {
        private const val serialVersionUID = 1L
    }

    var id: Long = 0
    var text: String? = null
    var author: String? = null
    var timesShowed: Int = 0
    var language: String? = null

    init {
        id = -1
        timesShowed = 0
    }

}
